op-acceptance-tests/tests/isthmus/operator_fee/test_case_generator.go

```diff
@@ -0,0 +1,138 @@
+package isthmus
+
+import (
+	cryptoRand "crypto/rand"
+	"fmt"
+	"math"
+	"math/big"
+	"math/rand"
+)
+
+type TestParams struct {
+	ID                  string
+	OperatorFeeScalar   uint32
+	OperatorFeeConstant uint64
+	L1BaseFeeScalar     uint32
+	L1BlobBaseFeeScalar uint32
+}
+
+// Generated random values for broader test coverage
+
+func GenerateAllTestParamsCases(numGeneratedValues int, l2WalletBalance *big.Int) []TestParams {
```

Comment:
Two comments:
1. Do we really want to generate all permutations? That could be a LOT of tests cases.
2. Param `l2WalletBalance` seems unused?

Reply:
1. This is the main idea behind the test, to verify using fuzzing that nothing unexpected happens. Agreed that these will take a while to run, so to minimize the time, the framework is configured to only add a single random value to each dimension.
2. Good catch

Reply:
Fixed in 4ddba19a657fd0d4c7d7e54bbf25c7330081ec2a